tyk-pro/values.yaml

```diff
@@ -353,6 +362,9 @@ gateway:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+  securityContext:
+    runAsUser: 1000
```

Comment:
Sorry for my ignorance, what will the user's group be? (since we don't have `runAsGroup` set up here)